### ISSUE TITLE
Support for p.config-server (v 3.x)

### ISSUE THREAD

**Issue by atuldalvi** [amenezes/config-client#37]
The library is failing for 3.x config server service instances (p.config-server).

I tried your suggestion from issue #35 as below but it still does not work.

```python
from config.cf import CF
from config.cfenv import CFenv

cf = CF(cfenv=CFenv(vcap_service_prefix="p.config-server"))
cf.get_config()
```
The issue appears to be with the path supplied to glom for accessing credentials from vcap_services. Due to a dot in the service prefix name (p.config-server), glom expands path to the json properties incorrectly (like "p.config-server.0.credentials.uri") and is unable to read the value.

Updating `_format_vcap_path` function in cfenv.py as below would address the problem.
```python
from glom import Path

def _format_vcap_path(self, path: str) -> Any:
   path = path[1:]
   return Path(Path(self.vcap_service_prefix),Path.from_text(path))
```

**Comment by amenezes** [comment]
Thanks for the info @atuldalvi I will investigate this issue.

**Comment by amenezes** [comment]
@atuldalvi could you try use this version to confirm the issue fix: https://pypi.org/project/config-client/0.11.1a3/

**Comment by amenezes** [comment]
fixed in version [0.11.1](https://github.com/amenezes/config-client/releases/tag/0.11.1)
